### DISCUSSION TITLE
CSRF Token Mismatch

### DISCUSSION THREAD

**Discussion by danielyedaniel** [cypress-io/cypress#22848] [Questions and Help] [answered]
I am running some tests on a site and every time I try to save a value in an element, it throws an error:

![image](https://user-images.githubusercontent.com/86787503/179809976-5a481501-41ab-4ff6-a648-11cf598ff5fa.png)

Is there a configuration that I need in my config file to bypass this?

**Comment by stavares843** [comment, marked as answer]
hello! 👋 

have you tried

```
Cypress.Cookies.defaults({
    preserve: "csrftoken"
})
```

as suggested [here](https://stackoverflow.com/a/51632308)?

**Reply by danielyedaniel** [reply]
This helped a lot, thank you
